scenes/IngameScores.gd

```diff
@@ -0,0 +1,31 @@
+extends VBoxContainer
```

Comment:
We try to use the GDScript style guide throughout, which means the name for this file should be ingame_scores.gd. I know not all files in the project use that convention (yet), but please try to stick to it in new files.